tool/tctl/common/auth_command_test.go

```diff
@@ -512,6 +542,14 @@ func TestGenerateAppCertificates(t *testing.T) {
 			ClusterName: clusterNameStr,
 		})
 	require.NoError(t, err)
+	user, err := types.NewUser("testuser")
```

Comment:
nit: visually separate blocks

```suggestion

	user, err := types.NewUser("testuser")
```